contrib/windows/julia-manifest.xml

```diff
@@ -7,6 +7,11 @@
       </requestedPrivileges>
     </security>
   </trustInfo>
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity type='win32'  name='Microsoft.Windows.Common-Controls' version='6.0.0.0'     processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```

Comment:
Where does this `publicKeyToken` come from? This isn't a required attribute, so can be removed because it's only meaningful for a strongly named assembly, which we don't generate.